src/Components/SearchBar.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+var text = "practise";
+class SearchBar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  render() {
+    return (
+      <div className="SearchBar">
+        <header className="app__header">
+          <section class="search">
+            <form action="">
```

Comment:
You don't actually need a form to contain these inputs, as you'll have custom javascript to deal with the interactions with those inputs.